mixer/test/client/env/http_client.go

```diff
@@ -85,7 +85,9 @@ func HTTPPost(url string, contentType string, reqBody string) (code int, respBod
 	}
 	respBody = string(body)
 	code = resp.StatusCode
-	log.Println(respBody)
+	if len(respBody) < 10240 {
+		log.Println(respBody)
```

Comment:
you could also use fortio's

`fhttp.DebugSummary(respBody, 512)` will escape and safely print up to 512 bytes, and if more will put the first 256 and last 256

https://github.com/istio/fortio/blob/master/fhttp/http_utils.go#L188-L197